### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by raphink**
These tests make sure empty strings are parsed as empty slices.

### CODE REVIEW

File: env_test.go [hunk 1]
```diff
@@ -55,6 +55,10 @@ func TestEmptyVars(t *testing.T) {
 	assert.Equal(t, "", cfg.Some)
 	assert.Equal(t, false, cfg.Other)
 	assert.Equal(t, 0, cfg.Port)
+  assert.Equal(t, 0, len(cfg.Strings))
+  assert.Equal(t, 0, len(cfg.SepStrings))
+  assert.Equal(t, 0, len(cfg.Numbers))
+  assert.Equal(t, 0, len(cfg.Bools))
```
Comment (caarlos0) [inline env_test.go]: the indent level seems off, can you fix it?

**Comment by caarlos0** [comment]
thanks @raphink , one minor comment above :)

**Comment by raphink** [comment]
@caarlos0 fixed

**Comment by caarlos0** [comment]
Thanks @raphink :+1:

**Comment by raphink** [comment]
You're welcome. I actually wrote these tests to make sure passing an empty string resulting in an empty array, since `strings.Split()` returns an array containing a single empty string. The tests passed, so I'm guessing it's fine, although in my code before, I had to use `FieldsFunc()` to avoid that edge case…
